styles/Orgnjdge.module.css

```diff
@@ -0,0 +1,213 @@
+.pplcon {
+    width: 100%;
+    display: flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    padding-left: 3.1%;
+    flex-direction: column;
+    position: relative;
+    margin-bottom: 4.5%;
+}
+
+.hdn1 {
+    font-family: var(--title);
+    text-transform: capitalize;
+    background: linear-gradient(to bottom, var(--linearg1), var(--linearg2));
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    font-weight: normal;
+    font-style: normal;
+    line-height: 87px;
+    font-size: 70px;
+    letter-spacing: -0.032em;
+}
+
+.carousel {
+    width: 98%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.taicon {
+    font-size: 50px;
```

Comment:
Change font-size to 30px and subsequently in media queries

Reply:
You can refer to this [codesandbox](https://codesandbox.io/s/cool-tree-omcx4) to look at how the above changes will perform!